napari/_qt/qt_viewer.py

```diff
@@ -516,11 +516,11 @@ def add_to_console_backlog(self, variables):
             for name in vlist:
                 try:
                     vdict[name] = eval(name, cf.f_globals, cf.f_locals)
-                except:  # noqa: E722
-                    print(
+                except NameError as err:
+                    raise NameError(
                         f'Could not get variable {name} from '
                         f'{cf.f_code.co_name}'
-                    )
+                    ) from err
```

Comment:
Um, this is a major behavioral change? I think it needs to be reverted — not finding a variable should not crash the console. *Maybe* it could be a warning. But I think this was added because it happens relatively often and is benign.

Reply:
Yeah this is blocking for me, but I wanted to put it out and hear comments cause I'm not sure I understand the code... Mostly, I'm not sure how to test this. Is this even crashing the console? Or just raising an exception that shows up at the bottom right (which would be fine)?